### PR TITLE
pleroma: 2.5.0 -> 2.5.1

### PR DESCRIPTION
**Pull request by picnoir**
Bumping pleroma. Crypt landed in hex, we can remove the custom git
override.

The manually added git deps were manually double-checked.

Upstream changelog: https://git.pleroma.social/pleroma/pleroma/-/releases/v2.5.1###### Description of changes

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by NinjaTrappeur** [comment]
@GrahamcOfBorg test pleroma

Note: testing this on my personal instance, do not merge yet.

**Comment by NinjaTrappeur** [comment]
Deployed that in the wild. I don't see anything obviously wrong happening.

This is ready to be reviewed/merged from my side.

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 219304` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pleroma</li>
  </ul>
</details>

File: pkgs/servers/pleroma/mix.nix [hunk 1]
```diff
@@ -281,6 +281,19 @@ let
       beamDeps = [ ecto ];
     };
 
+    crypt = buildRebar3 rec {
+      name = "crypt";
```
Comment (superherointj) [inline pkgs/servers/pleroma/mix.nix:285]:
```suggestion
      pname = "crypt";
```

File: pkgs/servers/pleroma/mix.nix [hunk 2]
```diff
@@ -281,6 +281,19 @@ let
...
+      version = "1.0.1";
+
+      src = fetchHex {
+        pkg = "${name}";
```
Comment (superherointj) [inline pkgs/servers/pleroma/mix.nix:289]:
```suggestion
        pkg = pname;
```

File: pkgs/servers/pleroma/mix.nix [hunk 3]
```diff
@@ -281,6 +281,19 @@ let
...
+        version = "${version}";
```
Comment (superherointj) [inline pkgs/servers/pleroma/mix.nix:290]:
```suggestion
        version = version;
```

File: pkgs/servers/pleroma/mix.nix (hunk 2 above)
Comment (superherointj) [reply pkgs/servers/pleroma/mix.nix:289]: Is it automatically generated? If it is, then, I think something else should change.

File: pkgs/servers/pleroma/mix.nix (hunk 1 above)
Comment (superherointj) [reply pkgs/servers/pleroma/mix.nix:285]: Is it automatically generated? If it is, then, I think something else should change.

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 219304` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pleroma</li>
  </ul>
</details>

**Pull request merged**